extra/debian/control

```diff
@@ -29,6 +29,7 @@ Build-Depends: curl,
                python-gobject,
                python-jinja2,
                python-lxml,
+               python-lzma,
```

Comment:
Fedora 21 and 20 at least already install `lzma` as a part of their standard python installation:

    $ docker run -it fedora:20 python
    Python 2.7.5 (default, Jun 25 2014, 10:19:55) 
    [GCC 4.8.2 20131212 (Red Hat 4.8.2-7)] on linux2
    Type "help", "copyright", "credits" or "license" for more information.
    >>> import lzma
    >>> lzma.LZMAFile
    <type 'lzma.LZMAFile'>